x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/serverless_function_latency.ts

```diff
@@ -32,6 +32,13 @@ const chartBase: ChartBase = {
   type: 'linemark',
   yUnit: 'time',
   series: {},
+  description: i18n.translate(
+    'xpack.apm.agentMetrics.serverless.avgDuration.description',
+    {
+      defaultMessage:
+        "The Transaction Duration shows your serverless function’s latency to process the request and respond back to the client. The serverless runtime might be busy with additional tasks (e.g. flushing APM data) after your function's response to the client. This does not affect the Transaction Duration but contributes to the overall Billed Duration for your serverless function.",
```

Comment:
Has this line of copy been around @bmorelli25 ?

Reply:
```suggestion
        "Transaction duration is time spent processing and responding to a request. If the request is queued it will not be contribute to the transaction duration but will contribute the overall billed duration.",
```

Reply:
> Has this line of copy been around @bmorelli25 ?

I don't think so.